server/portal/apps/projects/workspace_operations/shared_workspace_operations.py

```diff
@@ -352,6 +353,10 @@ def get_project(client, workspace_id):
             access = 'edit'
         elif perms.permission == 'READ':
             access = 'read'
+        else:
+            logger.info(f"System shared to user without proper Tapis file permissions: {system_id}, username: {username}")
```

Comment:
Just curious. What cases are we catching for here?

Reply:
Ideally, it shouldn't be necessary. But when I added a user with tapipy and forgot to set the READ/MODIFY permissions, it caused the filelisting to break for everyone on the tapis system. And it was difficult to figure out why, because there wasn't any error messaging.

So, cases where either a user is manually updating systems, or the process to add a user is interrupted. In all, I was trying to make the code a little bit more robust, but I don't think this is an absolutely necessary part of the PR.

Reply:
Gotcha 👍